<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>744</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>11</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>9</y>
     <width>501</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="BubbleList" name="bubbl_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>481</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QPushButton" name="file_send">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>发送文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_button">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lable_speed">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="input_Edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>481</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="personList" name="friendList_widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>211</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEdit_search"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_addFriend">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BubbleList</class>
   <extends>QWidget</extends>
   <header>bubble.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>personList</class>
   <extends>QListWidget</extends>
   <header>personlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
